tests/integ/modin/series/test_value_counts.py

```diff
@@ -137,3 +180,123 @@ def test_series_value_counts(native_series, normalize, sort, ascending, dropna):
             normalize=normalize, sort=sort, ascending=ascending, dropna=dropna
         ),
     )
+
+
+@pytest.mark.parametrize(
+    "native_series, expected, normalize, sort, ascending, dropna",
+    [
+        (
+            SERIES1,
+            native_pd.Series(
+                [0.04, 0.12, 0.12, 0.12, 0.12, 0.16, 0.16, 0.16],
+                index=[8, 1, 5, 6, 7, 2, 3, 4],
+                name="proportion",
+            ),
+            True,
+            True,
+            True,
+            True,
+        ),
+        (
+            SERIES1,
+            native_pd.Series(
+                [0.04, 0.12, 0.12, 0.12, 0.12, 0.16, 0.16, 0.16],
+                index=[8, 1, 5, 6, 7, 2, 3, 4],
+                name="proportion",
+            ),
+            True,
+            True,
+            True,
+            False,
+        ),
+        (
+            SERIES1,
+            native_pd.Series(
+                [1, 3, 3, 3, 3, 4, 4, 4], index=[8, 1, 5, 6, 7, 2, 3, 4], name="count"
+            ),
+            False,
+            True,
+            True,
+            True,
+        ),
+        (
+            SERIES1,
+            native_pd.Series(
+                [1, 3, 3, 3, 3, 4, 4, 4], index=[8, 1, 5, 6, 7, 2, 3, 4], name="count"
+            ),
+            False,
+            True,
+            True,
+            False,
+        ),
+        (
+            SERIES2,
+            native_pd.Series(
+                [0.04, 0.12, 0.12, 0.12, 0.12, 0.16, 0.16, 0.16],
+                index=["h", "a", "e", "f", "g", "b", "c", "d"],
+                name="proportion",
+            ),
+            True,
+            True,
+            True,
+            True,
+        ),
+        (
+            SERIES2,
+            native_pd.Series(
+                [0.04, 0.12, 0.12, 0.12, 0.12, 0.16, 0.16, 0.16],
+                index=["h", "a", "e", "f", "g", "b", "c", "d"],
+                name="proportion",
+            ),
+            True,
+            True,
+            True,
+            False,
+        ),
+        (
+            SERIES2,
+            native_pd.Series(
+                [1, 3, 3, 3, 3, 4, 4, 4],
+                index=["h", "a", "e", "f", "g", "b", "c", "d"],
+                name="count",
+            ),
+            False,
+            True,
+            True,
+            True,
+        ),
+        (
+            SERIES2,
+            native_pd.Series(
+                [1, 3, 3, 3, 3, 4, 4, 4],
+                index=["h", "a", "e", "f", "g", "b", "c", "d"],
+                name="count",
+            ),
+            False,
+            True,
+            True,
+            False,
+        ),
+    ],
+)
+@sql_count_checker(query_count=1)
+def test_series_value_counts_non_deterministic_pandas_behavior(
+    native_series, expected, normalize, sort, ascending, dropna
+):
```

Comment:
@sfc-gh-vbudati sorry, i am little bit confused by this change here. i see you now added a new test here to directly check the result. Then how does this fix the other test point test_series_value_counts? the other test seems going to continue fail inside jenkins or even github under some cases, right?
The purpose here is to make sure out test are stable under out env, right? so we need to fix the test_series_value_counts with correct result check, right ?

Reply:
Sorry, I forgot to remove the original faulty tests - I just did that. Only tests that pass should be here. I only explicitly compare the results for the tests that have Jenkins failures - otherwise I'd have to do 80 comparisons.